microsoft/runbook/azure.yml

```diff
@@ -47,4 +47,4 @@ platform:
         marketplace: $(marketplace_image)
         vhd: $(vhd)
         location: $(location)
-        vm_size: $(vm_size)
+        vm_size: $(vm_size)
```

Comment:
@squirrelsc do you know why github is complaining about this line here?

Reply:
@bamuir there is a new line in the end, don't remove it